### DISCUSSION TITLE
angular/cant-bind-to-formgroup-since-it-is-not-a-known-property-of-form/

### DISCUSSION THREAD

**Discussion by giscus** [arungudelli/angularjswiki#166] [Announcements]
# angular/cant-bind-to-formgroup-since-it-is-not-a-known-property-of-form/

Steps to fix Can't bind to formGroup since it isn't a known property of form error in Angular,1. Import FormsModule, ReactiveFormsModule from @angular/forms in each and every module that uses FormGroup. 2. Add FormsModule and ReactiveFormsModule into imports array of your module.3. In component html file use formGroup selector to bind the form data

https://www.angularjswiki.com/angular/cant-bind-to-formgroup-since-it-is-not-a-known-property-of-form/

**Comment by MrMartinez** [comment]
i have the same problem with angular 13. the proyect has some subModules and import FormsModule and ReactiveFormsModule in every child components and not working.

**Reply by arungudelli** [reply]
Do you have any stackblitz demo link?
